index.html

```diff
@@ -0,0 +1,92 @@
+                    <li><a href="#">Портфоліо</a></li>
+                    <li><a href="#">Контакти</a></li>
+                </ul>
+            </nav>
+            
+            <div>
+                <a href="mailto:info@devstudio.com">info@devstudio.com</a>
+                <a href="tel:+38 096 111 11 11">+38 096 111 11 11</a>
+            </div>
+        </div>
+    </header>
+
+        <main>
+            <h1>Ефективні рішення для вашого бізнесу</h1>
+            <button type="button">Замовити послугу</button>
+    
+            <section class="services">
+                <div>
+                    <h3>Увага до деталей</h3>
+                    <p>Цінуємо дрібниці і нові підходи.</p>
+                </div>
+                <div>
+                    <h2>Пунктуальність</h2>
+                    <p>Завжди виконуємо роботу вчасно.</p>
+                </div>
+                <div>
+                    <h2>Планування</h2>
+                    <p>Тактика для успіху вашого бізнесу.</p>
+                </div>
+                <div>
+                    <h2>Сучасні технології</h2>
+                    <p>Впроваджуємо новітні технології для виконання складних задач.</p>
+                </div>
+            </section>
+        
+            <section class="portfolio">
+                <h2>Чим ми займаємося</h2>
+                <img src="images/whatwedo1.png" alt="Десктопні додатки" width="370" height="294">
+                <img src="images/whatwedo2.png" alt="Мобільні додатки" width="370" height="294">
+                <img src="images/whatwedo3.png" alt="Дизайнерські рішення" width="370" height="294">
+            </section>
+        
+            <section class="team">
+                <h2>Наша команда</h2>
+                <div class="team-members">
+                    <div>
+                        <img src="images/Ighor.png" alt="Ігор Дем'янченко" width="270" height="260">
+                        <p>Ігор Дем'янченко<br>Product Designer</p>
+                    </div>
+                    <div>
+                        <img src="images/olga.png" alt="Ольга Ренійка" width="270" height="260">
+                        <p>Ольга Ренійка<br>Frontend Developer</p>
+                    </div>
+                    <div>
+                        <img src="images/mukola.png" alt="Микола Тарасов" width="270" height="260">
+                        <p>Микола Тарасов<br>Marketing</p>
+                    </div>
+                    <div>
+                        <img src="images/muhaylo.png" alt="Михайло Єрмаков" width="270" height="260">
+                        <p>Михайло Єрмаков<br>UI Designer</p>
+                    </div>
+                </div>
+            </section>
+        </main>
+    
+        <footer>
+            <img src="images/WebStudio.jpg" alt="Footer logo" width="145" height="31">
+            <a href="https://maps.google.com/?q=пр-т%20Лесі%20Українки%2026%2C%20Київ" target="_blank">м. Київ, пр-т Лесі Українки, 26</a>
```

Comment:
"rel noopener rel noreferrer" are HTML attributes that can be used when adding links to external and internal pages to prevent security issues. They are simple to use, but provide additional protection against COR attacks and can also help users avoid links in their Browser to open.